web/_js/main.js

```diff
@@ -354,7 +354,7 @@ async function init(){
 
 		touchstart(e);
 
-	});
+	},	{passive: true});
```

Comment:
what do these do?

Reply:
Make lighthouse happy and less strain on older browsers